core/arch/arm/mm/pgt_cache.c

```diff
@@ -204,30 +217,38 @@ static struct pgt *pop_from_cache_list(vaddr_t vabase, void *ctx)
 	return p;
 }
 
+static uint16_t get_num_used_entries(struct pgt *pgt __maybe_unused)
+{
+#ifdef CFG_PAGED_USER_TA
+	return pgt->num_used_entries;
+#else
+	return 0;
+#endif
+}
+
 static struct pgt *pop_least_used_from_cache_list(void)
 {
 	struct pgt *pgt;
 	struct pgt *p_prev = NULL;
 	size_t least_used;
+	size_t next_used;
```

Comment:
`= 0;` maybe a append fixup commit to the series to fix the 2 missing init values above?